tests/thickness/test_thickness_gamma.py

```diff
@@ -0,0 +1,262 @@
+        'y': 2944.223069901633608,
+        'strike2': 273.00,
+        'dip_2': 46.00,
+        'id': 408.00,
+        'sf': 's0',
+    },
+]
+for row in structures:
+    row['geometry'] = shapely.Point(row['x'], row['y'])
+    del row['x'], row['y']
+
+structures = geopandas.GeoDataFrame(structures, crs='epsg:7854')
+
+faults = geopandas.GeoDataFrame(columns=['geometry'], crs='epsg:7854')
+
+path = tempfile.mkdtemp()
+
+bounding_box = {"minx": 0, "miny": 0, "maxx": 10000, "maxy": 10000, "base": 0, "top": -5000}
+
+create_raster(
+    os.path.join(path, "DEM.tif"),
+    (bounding_box['minx'], bounding_box['miny'], bounding_box['maxx'], bounding_box['maxy']),
+    7854,
+    1000,
+)
+
+geology.to_file(os.path.join(path, "geology.shp"))
+structures.to_file(os.path.join(path, "structures.shp"))
+faults.to_file(os.path.join(path, "faults.shp"))
+
+loop_project_filename = os.path.join(path, "local_source.loop3d")
+
+config = {
+    "structure": {
+        "orientation_type": "strike",
+        "dipdir_column": "strike2",
+        "dip_column": "dip_2",
+        "description_column": "DESCRIPTION",
+        "bedding_text": "Bed",
+        "overturned_column": "structypei",
+        "overturned_text": "BEOI",
+        "objectid_column": "objectid",
+        "desciption_column": "feature",
+    },
+    "geology": {
+        "unitname_column": "UNITNAME",
+        "alt_unitname_column": "UNITNAME",
+        "group_column": "GROUP",
+        "supergroup_column": "supersuite",
+        "description_column": "descriptn",
+        "minage_column": "min_age_ma",
+        "maxage_column": "max_age_ma",
+        "rocktype_column": "rocktype1",
+        "alt_rocktype_column": "rocktype2",
+        "sill_text": "sill",
+        "intrusive_text": "intrusive",
+        "volcanic_text": "volcanic",
+        "objectid_column": "ID",
+        "ignore_codes": ["cover"],
+    },
+    "fault": {
+        "structtype_column": "feature",
+        "fault_text": "Fault",
+        "dip_null_value": "0",
+        "dipdir_flag": "num",
+        "dipdir_column": "dip_dir",
+        "dip_column": "dip",
+        "orientation_type": "dip direction",
+        "dipestimate_column": "dip_est",
+        "dipestimate_text": "gentle,moderate,steep",
+        "name_column": "name",
+        "objectid_column": "objectid",
+    },
+    "fold": {
+        "structtype_column": "feature",
+        "fold_text": "Fold axial trace",
+        "description_column": "type",
+        "synform_text": "syncline",
+        "foldname_column": "NAME",
+        "objectid_column": "objectid",
+    },
+}
+
+
+proj = Project(
+    geology_filename=os.path.join(path, "geology.shp"),
+    fault_filename=os.path.join(path, "faults.shp"),
+    fold_filename=os.path.join(path, "faults.shp"),
+    structure_filename=os.path.join(path, "structures.shp"),
+    dtm_filename=os.path.join(path, 'DEM.tif'),
+    clut_filename=r'C:\Users\angel\Documents\GitHub\map2loop\map2loop\_datasets\clut_files\WA_clut.csv',
```

Comment:
this won't work on the build server

Reply:
fixed with https://github.com/Loop3D/map2loop/pull/59/commits/921182667babe99bca6e1bed3df1ef7dac073504

Reply:
in https://github.com/Loop3D/map2loop/pull/59/commits/921182667babe99bca6e1bed3df1ef7dac073504